web/themes/custom/stormy/templates/content/node--city-service.html.twig

```diff
@@ -106,19 +108,24 @@
     <div class="tab-content" id="serviceModesContent">
       {% for key, item in content.field_service_mode if key|first != '#' %}
         <div class="tab-pane fade{% if loop.first %} show active{% endif %}" id="{{ loop.index }}" role="tabpanel" aria-labelledby="{{ loop.index }}-tab">
-          {% if item['#paragraph'].field_time_to_complete.value is not empty %}
-            <p class="time-to-complete">{{ item['#paragraph'].field_time_to_complete.value }}</p>
+          {% if item['#paragraph'].field_step_by_step_instructions|length == 1 %}
+            {% for step in item['#paragraph'].field_step_by_step_instructions %}
```

Comment:
You can do this instead if you want to avoid having to loop over a single item:

```twig
<div class="onestep">{{ drupal_entity('paragraph', item['#paragraph'].field_step_by_step_instructions.0.target_id) }}</div>
```

Reply:
Let me see if Tracey passes the QA or not. If she does, I may go ahead with the extra loop for now and write a refactor story.